Update diagram custom docs (#414)
DevExpress/devextreme-documentation: concepts/05 Widgets/Diagram/20 Custom Shapes/20 Shapes with Custom Background Images.md

Use the [backgroundImageUrl](/api-reference/10%20UI%20Widgets/dxDiagram/1%20Configuration/customShapes/#backgroundImageUrl '/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/customShapes/#backgroundImageUrl') option to specify a background image for a shape.

> [note] Shape images should be supplied as SVG files. 

Reviewer: ```suggestion
[note] Shape images should be in SVG format. 
```
